apps/studio/pages/project/[ref]/advisors/query-performance.tsx

```diff
@@ -36,7 +36,7 @@ const QueryPerformanceReport: NextPageWithLayout = () => {
   })
 
   const config = PRESET_CONFIG[Presets.QUERY_PERFORMANCE]
-  const hooks = queriesFactory(config.queries, projectRef ?? 'default')
+  const hooks = queriesFactory(config.queries, project?.ref ?? 'default')
```

Comment:
any reason why we can't use ref from params? its faster than depending on useProjectContext

Reply:
i'll revert for now but lemme know if i might be overlooking something here